recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - setuptools
     - wheel
   run:
+    - s5cmd
```

Comment:
This [`s5cmd`](https://anaconda.org/conda-forge/s5cmd/files) package on conda-forge is the Golang package. Need to package the Python bindings at https://pypi.org/project/s5cmd to continue with this PR.